content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -38,51 +38,42 @@
 
 <blockquote class="commentary">
 
-  <p>WebRTC enables peer to peer communication.</p>
+  <p>WebRTC enables peer-to-peer communication, but it still needs servers so that clients can exchange metadata to coordinate communication through a process called signaling, and to cope with network address translators (NATs) and firewalls.</p>
 
-  <p>BUT...</p>
+<p>This article shows you how to build a signaling service, and how to deal with the quirks of real-world connectivity with STUN and TURN servers. It also explains how WebRTC apps can handle multiparty calls and interact with services, such as VoIP and PSTN (also known as telephones).</p>
 
-  <p>WebRTC still needs servers:</p>
-
-  <ul>
-    <li>For clients to exchange metadata to coordinate communication: this is called signaling.</li>
-    <li>To cope with network address translators (NATs) and firewalls.</li>
-  </ul>
-
-<p>In this article we show you how to build a signaling service, and how to deal with the quirks of real-world connectivity by using STUN and TURN servers. We also explain how WebRTC apps can handle multi-party calls and interact with services such as VoIP and PSTN (aka telephones).</p>
-
-<p>If you're not familiar with the basics of WebRTC, we strongly recommend you take a look at <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/">Getting Started With WebRTC</a> before reading this article.</p>
+<p>If you're not familiar with the basics of WebRTC, see <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/">Getting Started With WebRTC</a> before you read this article.</p>
 
 </blockquote>
 
 <h2 id="what-is-signaling">What is signaling?</h2>
 
-<p>Signaling is the process of coordinating communication. In order for a WebRTC application to set up a 'call', its clients need to exchange information:</p>
+<p>Signaling is the process of coordinating communication. In order for a WebRTC app to set up a call, its clients need to exchange information, such as the following:</p>
```

Comment:
I think you should probably change this to 'exchange the following information:'.

(This is the only information that's exchanged, i.e. it's not 'such as'.)

Reply:
Will fix

Reply:
Will fix